community/blog/2020-05-Azure-Dependency-Tracking-in-Observability-v0.2.0/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure dependency tracking in Observability v0.2.0
+
+With the release of the next version of [Arcus.Observability v0.2.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greattly expanded.
```

Comment:
I would add a paragraph before this referring to our previous post

Reply:
Added a sentance before it.